editor/src/components/canvas/canvas-strategies/absolute-reparent-to-flex-strategy.spec.browser2.tsx

```diff
@@ -404,4 +458,134 @@ describe('Absolute Reparent To Flex Strategy', () => {
   </div>`),
     )
   })
+  it('cannot reparent a generated element', async () => {
+    const renderResult = await renderTestEditorWithCode(
+      makeTestProjectCodeWithSnippet(defaultTestCode),
+      'await-first-dom-report',
+    )
+
+    const generatedAbsolutechildren = await renderResult.renderedDOM.findAllByTestId(
+      'generatedabsolutechild',
+    )
+    const absoluteChild = generatedAbsolutechildren[0]
+    const absoluteChildRect = absoluteChild.getBoundingClientRect()
+    const absoluteChildCenter = {
+      x: absoluteChildRect.x + absoluteChildRect.width / 2,
+      y: absoluteChildRect.y + absoluteChildRect.height / 2,
+    }
+    const firstFlexChild = await renderResult.renderedDOM.findByTestId('flexchild1')
+    const firstFlexChildRect = firstFlexChild.getBoundingClientRect()
+    const firstFlexChildCenter = {
+      x: firstFlexChildRect.x + firstFlexChildRect.width / 2,
+      y: firstFlexChildRect.y + firstFlexChildRect.height / 2,
+    }
+
+    await renderResult.getDispatchFollowUpActionsFinished()
+
+    const dragDelta = windowPoint({
+      x: firstFlexChildCenter.x - absoluteChildCenter.x,
+      y: firstFlexChildCenter.y - absoluteChildCenter.y,
+    })
+    await act(() => dragElement(renderResult, 'generatedabsolutechild', dragDelta, cmdModifier))
+
+    await renderResult.getDispatchFollowUpActionsFinished()
+
+    expect(getPrintedUiJsCode(renderResult.getEditorState())).toEqual(
+      makeTestProjectCodeWithSnippet(`
```

Comment:
It's unclear what (if anything) has changed here from the starting code. Has anything changed?

Reply:
Nothing should have changed in this case.

Reply:
In that case can you change this to compare the result against the starting code please so that that is explicit?

Reply:
Fixed in 71215f224.